psycop/common/feature_generation/sequences/test_patient_loaders.py

```diff
@@ -1,20 +1,51 @@
+import pandas as pd
+import polars as pl
+from polars.testing import assert_frame_equal
+
 from psycop.common.feature_generation.sequences.patient_loaders import (
     DiagnosisLoader,
+    keep_if_min_n_visits,
 )
 from psycop.common.test_utils.str_to_df import str_to_pl_df
 
 
-def test_diagnosis_formatting():
+def test_diagnosis_preprocessing():
     df = str_to_pl_df(
         """dw_ek_borger,datotid_slut,diagnosegruppestreng
     1,2023-01-01,A:DF431                    # Keep all, even though doesn't end with a hashtag
-    1,2023-01-01,A:DF431#+:ALFC3#B:DF329    # Keep up until, but not including, first hashtag
+    1,2023-01-01,A:DF431                    # Duplicate, do not keep
+    2,2023-01-01,A:DF439#+:ALFC3#B:DF329    # Keep up until, but not including, first hashtag
```

Comment:
We want a test showing that it splits correctly on e.g. "HCO:DF329", which was the problem you solved by rewriting the formatting :-)